inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/group/InlongGroupInfo.java

```diff
@@ -79,11 +79,20 @@ public class InlongGroupInfo {
     @ApiModelProperty(value = "Pulsar service URL")
     private String pulsarServiceUrl;
 
-    @ApiModelProperty(value = "Need zookeeper support, 0 false 1 true")
-    private Integer zookeeperEnabled;
+    @ApiModelProperty(value = "Whether to enable zookeeper? 0: disable, 1: enable")
```

Comment:
tubeMaster,pulsarAdminUrl,pulsarServiceUrl is non-relationship with inlong dataflow.
These are the params of inlong_cluster(TUBE,Pulsar).

Reply:
This logic involves many places, and I will modify it together with the naming of inongGroupId in another feature.

Reply:
Only manager-clients use these logic.
![image](https://user-images.githubusercontent.com/8925507/168545178-29ece1fb-6316-40cd-8e54-5d6de9a369fd.png)

Reply:
Not yet, the dashboard also needs those configs.